packages/app/src/systems/Settings/hooks/useSettings.ts

```diff
@@ -0,0 +1,65 @@
+import { useInterpret, useSelector } from '@xstate/react';
+import { useNavigate } from 'react-router-dom';
+
+import type { SettingsMachineState } from '../machines/settingsMachine';
+import { settingsMachine } from '../machines/settingsMachine';
+
+import type { AccountInputs } from '~/systems/Account';
+import { useAccount } from '~/systems/Account';
+import { Pages } from '~/systems/Core';
+
+const selectors = {
+  isChangingPassword: (state: SettingsMachineState) =>
+    state.matches('changingPassword'),
```

Comment:
Try to always use return here and avoid line spaces between object props (like in the rest of the codebase)